rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -321,7 +333,7 @@ def test_edit_manually_tracked_balances(rotkehlchen_api_server):
     response = requests.get(
         api_url_for(
             rotkehlchen_api_server,
-            'manuallytrackedbalancesresource',
+            "manuallytrackedbalancesresource",
```

Comment:
Okay I will stop here. Just undo all these please

```suggestion
            'manuallytrackedbalancesresource',
```